salt/modules/nacl.py

```diff
@@ -253,6 +252,14 @@ def keygen(sk_file=None, pk_file=None):
         salt-call nacl.keygen sk_file=/etc/salt/pki/master/nacl pk_file=/etc/salt/pki/master/nacl.pub
         salt-call --local nacl.keygen
     '''
+    if 'keyfile' in kwargs:
+        salt.utils.versions.warn_until(
+            'Fluorine',
+            'The \'keyfile\' argument has been deprecated and will be removed in Salt '
+            '{version}. Please use \'sk_file\' argument instead.'
+        )
+        sk_file = kwargs['sk_file']
```

Comment:
should this not be kwargs['keyfile'] ? if assume the idea to to warn and keep the old stuff working until Flourine.